seed/seed20_test.go

```diff
@@ -3243,9 +3242,12 @@ func (s *seed20Suite) writeValidAutoImportAssertion(c *C, sysLabel string, perm
 
 func (s *seed20Suite) writeInvalidAutoImportAssertion(c *C, sysLabel string, perm os.FileMode) {
 	autoImportAssert := filepath.Join(s.SeedDir, "systems", sysLabel, "auto-import.assert")
-	// write random data
+	// write pseudo-random data
 	randomness := make([]byte, 512)
-	rand.Read(randomness)
+	const numLetters = 'z' - 'a' + 1
+	for i, _ := range randomness {
+		randomness[i] = byte('a' + i % numLetters)
+	}
```

Comment:
This looks good, just one minor nitpick. `randomness` will always have the same data now. Since it's not random anymore, we could just simplify to:
```suggestion
	err := ioutil.WriteFile(autoImportAssert, []byte("invalid"), perm)
```

It's not 512 bytes but that seems liked an arbitrary choice so I think it's fine

Reply:
Right, let's see if @kubiko had a reason to fix this to 512 or if your suggestion is enough to test the intended code paths.

Reply:
Silly question - we have `randutil.RandomString()` - would that fit here?

Reply:
You could also do:
```suggestion
	err := ioutil.WriteFile(autoImportAssert, []byte(strings.Repeat("a", 512)), perm)
```

Reply:
I agree that it shouldn't be completely random for a unit test, since that destroys the reproducibility. Just generate 512 bytes of random garbage and hardcode that insteads :)

Reply:
I went for this last suggestion, thanks!